specification/kubernetesconfiguration/resource-manager/Microsoft.KubernetesConfiguration/preview/2024-04-01-preview/fluxconfiguration.json

```diff
@@ -0,0 +1,1822 @@
+            "type": "string"
+          }
+        },
+        "matchOIDCIdentity": {
+          "description": "Array defining the criteria for matching the OIDC identity while verifying an OCI artifact.",
+          "type": "array",
+          "items": {
+            "type": "object",
+            "$ref": "#/definitions/MatchOidcIdentityPatchDefinition"
+          },
+          "x-ms-identifiers": [],
+          "x-nullable": true
+        }
+      }
+    },
+    "MatchOidcIdentityDefinition": {
+      "description": "MatchOIDCIdentity defines the criteria for matching the identity while verifying an OCI artifact.",
+      "type": "object",
+      "x-nullable": true,
+      "properties": {
+        "issuer": {
+          "description": "The regex pattern to match against to verify the OIDC issuer.",
+          "type": "string"
+        },
+        "subject": {
+          "description": "The regex pattern to match against to verify the identity subject.",
+          "type": "string"
+        }
+      }
+    },
+    "MatchOidcIdentityPatchDefinition": {
+      "description": "MatchOIDCIdentity defines the criteria for matching the identity while verifying an OCI artifact.",
+      "type": "object",
+      "x-nullable": true,
+      "properties": {
+        "issuer": {
+          "description": "The regex pattern to match against to verify the OIDC issuer.",
+          "type": "string"
+        },
+        "subject": {
+          "description": "The regex pattern to match against to verify the identity subject.",
+          "type": "string"
+        }
+      }
+    },
+    "PostBuildDefinition": {
+      "description": "The postBuild definitions defining variable substitutions for this Kustomization after kustomize build.",
+      "type": "object",
+      "x-nullable": true,
+      "properties": {
+        "substitute": {
+          "description": "Key/value pairs holding the variables to be substituted in this Kustomization.",
+          "type": "object",
+          "x-nullable": true,
+          "additionalProperties": {
+            "type": "string"
+          }
+        },
+        "substituteFrom": {
+          "description": "Array of ConfigMaps/Secrets from which the variables are substituted for this Kustomization.",
+          "type": "array",
+          "items": {
+            "type": "object",
+            "$ref": "#/definitions/SubstituteFromDefinition"
+          },
+          "x-ms-identifiers": [],
+          "x-nullable": true
+        }
+      }
+    },
+    "PostBuildPatchDefinition": {
+      "description": "The postBuild definitions defining variable substitutions for this Kustomization after kustomize build.",
+      "type": "object",
+      "x-nullable": true,
+      "properties": {
+        "substitute": {
+          "description": "Key/value pairs holding the variables to be substituted in this Kustomization.",
+          "type": "object",
+          "x-nullable": true,
+          "additionalProperties": {
```

Comment:
>additionalProperties [](http://example.com/codeflow?start=11&length=20)

This is an ARM anti-pattern. Please schematize this object or model it as a collection.

Reply:
please look at above comment left